internal/rbd/nodeserver.go

```diff
@@ -687,7 +687,7 @@ func getDevicePath(ctx context.Context, volumePath string) (string, error) {
 	if err != nil {
 		klog.Errorf(util.Log(ctx, "failed to find image metadata: %v"), err)
 	}
-	device, found := findDeviceMappingImage(ctx, imgInfo.Pool, imgInfo.ImageName, imgInfo.NbdAccess)
+	device, found := findDeviceMappingImage(ctx, imgInfo.Pool, imgInfo.Namespace, imgInfo.ImageName, imgInfo.NbdAccess)
```

Comment:
TODO: `findDeviceMappingImage()` should probably become `imgInfo.findDeviceMappingImage()`

This can be placed in its own commit, or even its own PR

Reply:
`rbdVolume` uses the `findDeviceMappingImage` [here](https://github.com/ceph/ceph-csi/pull/1035/files#diff-9e4cf5c8370e4beee82e84b9818a53acL146).

should I implement it for both of them or just leave it like this?

Reply:
Not needed for now, it is something that might cleanup the code and improve maintenance over time.